src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -0,0 +1,268 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert ViT checkpoints from the timm library."""
+
+
+import argparse
+from pathlib import Path
+
+import torch
+from PIL import Image
+from torchvision.transforms import Compose, Normalize, Resize, ToTensor
+
+import requests
+import timm
+from transformers import ViTConfig, ViTFeatureExtractor, ViTForImageClassification
+from transformers.utils import logging
+from transformers.utils.imagenet_classes import id2label
+
+
+logging.set_verbosity_info()
+logger = logging.get_logger(__name__)
+
+
+# here we list all keys to be renamed (original name on the left, our name on the right)
+def create_rename_keys(config, base_model=False):
+    rename_keys = []
+    for i in range(config.num_hidden_layers):
+        # encoder layers: output projection, 2 feedforward neural networks and 2 layernorms
+        rename_keys.append(
+            ("blocks." + str(i) + ".norm1.weight", "vit.encoder.layer." + str(i) + ".layernorm_before.weight")
```

Comment:
```suggestion
            (f"blocks.{i}.norm1.weight", f"vit.encoder.layer.{i}.layernorm_before.weight")
```
A f-string is more readable here. Use the same for all rename_key down there.